app/services/container_topology_service.rb

```diff
@@ -89,22 +89,26 @@ def entity_display_type(entity)
     end
   end
 
-  def build_entity_data(entity)
+  def entity_id(entity)
     type = entity_type(entity)
-    status = entity_status(entity, type)
-
     if entity.kind_of?(ManageIQ::Providers::ContainerManager)
       id = entity.id.to_s
```

Comment:
> also, the id.to_s was not added as part of this patch, so I'm unclear why is this part of this particular change review

@abonas it was a curiosity when looking at the method as a whole.  I am more concerned with the hardcoded class names than this "string id".